platform/ios/app/MBXViewController.m

```diff
@@ -703,6 +708,22 @@ - (void)parseFeaturesAddingCount:(NSUInteger)featuresCount usingViews:(BOOL)useV
     });
 }
 
+- (void)animateAnnotationView
+    {
+        MGLPointAnnotation *annot = [[MGLPointAnnotation alloc] init];
+        annot.coordinate = self.mapView.centerCoordinate;
+        [self.mapView addAnnotation:annot];
+        
+        CGPoint point = CGPointMake(-200, CGRectGetMidY(self.view.frame));
```

Comment:
The magic value (-200) happens to move annotation views into the view reuse area outside of the viewport. If we change the size of annotation views in the future it may not. Would it be worth it to somehow special case this annotation to guarantee the size of the view? If not, it may at least be worth adding a comment here explaining why this value works today.